### ISSUE TITLE
Profile: Make last name mandatory

### ISSUE THREAD

**Issue by richarddabin** [nhsengland/futurenhs#353]
Currently when you entire you profile details, Last Name is labelled as optional. This should be a mandatory field and labelled as such.

**Comment by tim-hunt** [comment]
There have been discussions on this previously where we've been directed to make last name optional as having a last name isn't a certainty - https://en.wikipedia.org/wiki/Mononym#Mononym-normal. This has been a concious choice.

Can I advise we keep a decision log somewhere where things like this can be recorded and refered to to help prevent the same discussions/issues cropping up?

**Comment by simplybenuk** [comment]
Good point @tim-hunt, that didn't come up when we were discussing this.

Re decisions log, think that will quickly get unusable, as long as we are recording these things either here or on our Azure tickets so we can look back on them that should be fine.

**Comment by simplybenuk** [comment]
How about we change the format to Full Name as that will accommodate everyone and then we just have one mandatory field 

![Screenshot_20220910-075823.png](https://user-images.githubusercontent.com/9471595/189473206-20a71be2-a2f1-4a6d-bcbf-2cb20b9607e3.png)

**Comment by richarddabin** [comment]
Is there a specific field that we need to fill for Last Name? Can we split a field which is Full Name into First and Last name and use that data appropriately? If we merged it would it break any other data we already have?

**Comment by simplybenuk** [comment]
Probably the easiest thing to do is just leave Last name as optional but remove the optional label which looks odd

**Comment by richarddabin** [comment]
Not sure it's good practice to remove the Optional label. Maybe leave it as it is until we can investigate what data we need and what we can do with it?

**Comment by simplybenuk** [comment]
Agree. It's not a priority 

My comment around removing the optional term was reading between the lines on the [gov design system](https://design-system.service.gov.uk/patterns/names/), where for the optional middle name they say:

_"The label should not include (optional). Users will enter their middle names if they have them and skip the field if they do not."_

I felt we could apply this to the last name too
